### ISSUE TITLE
Laravel 9 or PHP 8 support

### ISSUE THREAD

**Issue by jcsoriano** [tightenco/nova-google-analytics#44]
First, thank you for sharing this great package!

Currently I am checking if the usual packages I include in new projects are compatible with Laravel 9. And I encountered this error when installing your package:

```
  Problem 1
    - tightenco/nova-google-analytics[v1.0.0, ..., v1.0.1] require spatie/laravel-analytics ^3.6 -> satisfiable by spatie/laravel-analytics[3.6.0, ..., 3.11.0].
    - spatie/laravel-analytics[3.6.0, ..., 3.6.x-dev] require php ^7.0 -> your php version (8.0.10) does not satisfy that requirement.
    - spatie/laravel-analytics[3.7.0, ..., 3.10.1] require php ^7.2 -> your php version (8.0.10) does not satisfy that requirement.
    - spatie/laravel-analytics[3.10.2, ..., 3.11.0] require laravel/framework ^6.0|^7.0|^8.0 -> found laravel/framework[v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev] but it conflicts with your root composer.json require (^9.0).
    - Root composer.json requires tightenco/nova-google-analytics ^1.0 -> satisfiable by tightenco/nova-google-analytics[v1.0.0, v1.0.1].

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
```

## Steps to replicate:

1. `laravel new l9` and install Nova
2. `composer require tightenco/nova-google-analytics`

The same error occurs even when I include `--with-all-dependencies`

**Comment by krievley** [comment]
@jcsoriano - Thank you for reporting this. I have a current branch open to add support for PHP 8 and Laravel 9 and I will tag you when it is released. Thanks again for your interest in this package.

**Comment by NowakAdmin** [comment]
updating to spatie/laravel-analytics v 4.1.0 should done things (already upgraded to laravel 9 and php 8)

https://github.com/spatie/laravel-analytics/releases/tag/4.1.0

**Comment by krievley** [comment]
@jcsoriano @NowakAdmin -- The support for PHP 8 and Laravel 9 has been released as [v2.0.0](https://github.com/tighten/nova-google-analytics/releases/tag/v2.0.0).
